packages/lib-react-components/src/ZooHeader/ZooHeader.js

```diff
@@ -5,34 +5,35 @@ import { useResizeDetector } from 'react-resize-detector'
 import styled from 'styled-components'
 import { getHost } from './helpers'
 import { useTranslation } from '../translations/i18n'
+import { useHasMounted } from '../hooks'
 
 import MainNavList from './components/MainNavList'
-import SignedInUserNavigation from './components/SignedInUserNavigation'
-import SignedOutUserNavigation from './components/SignedOutUserNavigation'
+import NarrowMainNavMenu from './components/NarrowMainNavMenu'
+import UserNavigation from './components/UserNavigation/UserNavigation.js'
 import ZooniverseLogo from '../ZooniverseLogo'
 
 export const StyledHeader = styled(Box)`
-  color: #B2B2B2;
+  color: #b2b2b2;
```

Comment:
Can this color be pulled into / out of the ZooTheme? It's being done on line 27 of this file so that would seem pretty doable.

Might also be something to flag with Sean about consistent application of colors for things like this (it's so easy to litter the codebase with colors and other styles).

Reply:
This color is used throughout ZooHeader, but you're right it's not in the Grommet theme object. I have it on my todo's to ask Sean about adding it as a named themed color

Reply:
@goplayoutside3 Just seeing this discrepancy. I would vote for using already-defined colors such as Grey 4 #CBCCCB or Grey 5 #A6A7A9 from our design system.